### DISCUSSION TITLE
Tacotron2 Test audios 1s blips

### DISCUSSION THREAD

**Discussion by epicstar7** [coqui-ai/TTS#1262] [General Q&A]
After running 40K steps on 270audio files with Tacotron2 the Testaudios in Tensorboard are still <1s blips. Is this behavior to be expected or does this mean something is wrong, e.g. config settings? Not sure if I shouldn't continue training with these results...

Thanks!

**Comment by domcross** [comment]
270 audio files is quite low number, can you use more?
Training a Taco2 model with a decent dataset (several 1000 audio files) should give you first audible results around 10k steps.

**Reply by epicstar7** [reply]
I cannot get more than these 270 recordings sadly. I will probably need to train a long long time to get some results here. Do you have any tips or suggestions on more efficient training methods or models with these amounts of samples or is Taco2 the best option anyway?

**Reply by kin0303** [reply]
> 270 audio files is quite low number, can you use more? Training a Taco2 model with a decent dataset (several 1000 audio files) should give you first audible results around 10k steps.

Hi, can I see your config when you are training using tacotron2?
